docs/tls/README.md

```diff
@@ -1,38 +1,37 @@
 # How to secure access to Minio server with TLS [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
 
-In this document, we will configure Minio servers with TLS certificates for both Linux and Windows.
+This document explains how to configure Minio server with TLS certificates on Linux and Windows platforms.
 
 ## 1. Prerequisites
 
-* Download Minio server from [here](https://docs.minio.io/docs/minio-quickstart-guide)
+Download Minio server from [here](https://docs.minio.io/docs/minio-quickstart-guide)
 
-## 2. Configure with existing certificates
+## 2. Existing certificates
 
-Assuming that you are already having private and public certificates, you will need to copy them under `certs` in your Minio config directory using the names `private.key` and `public.crt` for key and public certificates respectively.
+If you have already acquired private and public certificates, copy them under `certs` directory in your Minio config directory (by default this is `/home/username/.minio/`). Note that the file names should be `private.key` and `public.crt` for key and public certificate respectively.
```

Comment:
We can say about windows home path as well? like we can say 

- `${HOME}/.minio/` 
- `%%USERPROFILE%%\.minio\` 

?

Reply:
sure, will do

Reply:
this is done